backend/src/ee/services/audit-log/audit-log-types.ts

```diff
@@ -2720,6 +2746,15 @@ interface OrderCertificateFromProfile {
   };
 }
 
+interface RenewCertificate {
+  type: EventType.RENEW_CERTIFICATE;
+  metadata: {
+    originalCertificateId: string;
+    newCertificateId: string;
+    profileName: string;
+  };
+}
+
```

Comment:
IGNORE PLACEMENT OF COMMENT

Upon enabling auto-renewal again... it defaults to "20" for me but I can't actually save the configuration... should we be defaulting it to whatever's defined in the linked profile instead? 

<img width="1512" height="851" alt="image" src="https://github.com/user-attachments/assets/508e9788-993a-4249-b386-98012a7be25a" />

<img width="1512" height="848" alt="image" src="https://github.com/user-attachments/assets/dbdf7dbf-d673-4120-8619-7a8be755357d" />